config/BUILD.bazel

```diff
@@ -34,22 +34,35 @@ controller_gen(
     visibility = ["//visibility:public"],
 )
 
-# Produces a provider components manifest without image versioning
-# To be used as a base for release and developer manifests
+filegroup(
+    name = "kustomize-yaml",
+    srcs = glob([
+        "crds/*.yaml",
+        "rbac/*.yaml",
+        "manager/*.yaml",
+        "default/*.yaml",
+    ]),
+    visibility = ["//visibility:public"],
+)
+
 genrule(
     name = "provider-non-stamped-base",
     srcs = [
-        "//config/default:kustomization.yaml",
-        "//config/default:rbac_role_binding_patch.yaml",
-        "//config/manager:manager.yaml",
+        "//config:default/rbac_role_binding_patch.yaml",
+        "//config:manager/manager.yaml",
         "config",
+        "//config:default/kustomization.yaml",
+        #"//config:rbac/kustomization.yaml",
```

Comment:
Is the comment here intentional?

Reply:
It suppose to be removed. Fixed in the new commit.